platform/mds/mds/src/main/java/org/motechproject/mds/query/PropertyBuilder.java

```diff
@@ -55,6 +55,9 @@ public static Property create(String name, Object value, String type, String ope
         } else if (value instanceof Range) {
             Range range = (Range) value;
             return new RangeProperty<>(name, range, type);
+        } else if (type.equals("java.lang.String") && StringUtils.isNotBlank(operator) && operator
```

Comment:
add this condition below into switch case statement and please use String.class.getName() method on the left side (raw string should always be on the left side for example : "test".equals(testVar)